pom.xml

```diff
@@ -926,7 +930,9 @@
             <forkCount>1</forkCount>
             <reuseForks>false</reuseForks>
             <forkedProcessTimeoutInSeconds>900</forkedProcessTimeoutInSeconds>
-            <argLine>-Xmx1024m -XX:+HeapDumpOnOutOfMemoryError</argLine>
+            <argLine>-Xmx4096m -XX:+HeapDumpOnOutOfMemoryError</argLine>
```

Comment:
Strange. Do we need 4G vs. 1G in the case of JDK17 for the surefire processes? Do we have an explanation for that?

Reply:
This value i modified as compile and run with jdk17 it is taking more time for some of the tests and some tests are timeout. I will revert this value to "xmx1024m" and check once. if any no issue then i revert this value and modify the test case accordingly.

Reply:
I cannot see this anymore, I guess it's working with 1024, resolving this